modules/javafx.graphics/src/main/java/javafx/scene/layout/VBox.java

```diff
@@ -154,16 +154,17 @@
     private static final String VGROW_CONSTRAINT = "vbox-vgrow";
 
     /**
-     * Sets the vertical grow priority for the child when contained by an vbox.
-     * If set, the vbox will use the priority to allocate additional space if the
-     * vbox is resized larger than it's preferred height.
+     * Sets the vertical grow priority for the child when contained by a vbox.
+     * If set, the vbox will use the priority value to allocate additional space if the
+     * vbox is resized larger than its preferred height.
      * If multiple vbox children have the same vertical grow priority, then the
      * extra space will be split evenly between them.
      * If no vertical grow priority is set on a child, the vbox will never
-     * allocate it additional vertical space if available.
-     * Setting the value to null will remove the constraint.
+     * allocate it additional vertical space, if available.
```

Comment:
This isn't a new problem, but the following is awkward:

"If no vertical grow priority is set on a child, the vbox will never allocate it additional vertical space, if available."

If this is trying to say that the vbox will never allocate extra space for that child, then it should be reworded slightly. Maybe something like this?

"If no vertical grow priority is set on a child, the vbox will never allocate any additional vertical space for that child."

I note that HBox has the same awkward sentence.

Reply:
Your suggestions looks good, I tried not to rewrite too much initially.

Reply:
Yeah, the only reason I suggested it is that I had to read that sentence three times before I realized what it was trying to say.